build_all_wheels.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# build compiled wheel versions of pike with multiple different python versions
+
+# ensure system build requirements are met (see README.md)
+
+# any additional arguments supplied will be passed to the final
+# `setup.py bdist_wheel` command and may be used to upload the
+# packages to artifactory
+#  ./build_all_wheels.sh upload -r http://artifactory01.prod.sea1.west.isilon.com/artifactory/api/pypi/pypi-local
+
+# recommend the use of `pyenv` to install and activate
+# multiple python versions before running the script.
+
+set -euo pipefail
+
+# XXX: py3.9 not supported by internal artifactory
+versions=(python2 python3.6 python3.7 python3.8)
+
+for v in ${versions[@]}
+do
+    rm -rf .build/$v && \
+    virtualenv -p $v .build/$v && \
+    .build/$v/bin/pip install 'build' && \
+    .build/$v/bin/python -m build && \
+    .build/$v/bin/python setup.py bdist_wheel $*
```

Comment:
er... This is where we would _use_ `build`:
```suggestion
    .build/$v/bin/python -m build --outdir dist/ .
    .build/$v/bin/pip install twine
    .build/$v/bin/twine $* dist/*
```
e.g. https://github.com/dmtucker/verdoc/blob/fce4eaefcd29cb7e6da5853b95974e54ecff13e7/tox.ini#L28-L36

Reply:
ah, so as it's written here, we're essentially building it twice. once with `build` and once with `setup.py`?

Reply:
Yes, exactly... The `setup.py` call rebuilds and uploads.

Reply:
Encountered ISILONEAT-1456. Raised a different dependant user story ISILONQE-3348 to address this.

Reply:
Might be good to leave a comment about that here (since it looks a little funky to see the lone build command 😋)

Reply:
Ack